test/Certificate.test.ts

```diff
@@ -35,13 +35,10 @@ describe('Certificate', () => {
     await expect(
       market
         .connect(buyer)
-        [
-          'swapFromSupplier(address,address,uint256,address,uint256,uint8,bytes32,bytes32)'
-        ](
+        ['swap(address,address,uint256,uint256,uint8,bytes32,bytes32)'](
```

Comment:
you can just delete this test since we have one in foundry covering it [here](https://github.com/nori-dot-eco/contracts/blob/7046e8446201ffb20664b74dc152ea709308a17f/test/checkout.int.t.sol#L806)